toponetx/classes/combinatorial_complex.py

```diff
@@ -528,3 +776,412 @@ def clone(self) -> "CombinatorialComplex":
         for cell in self.cells:
             CC.add_cell(cell, self.cells.get_rank(cell))
         return CC
+
+    def is_connected(self, s=1, cells=False):
+        """Determine if combintorial complex is :term:`s-connected <s-connected, s-node-connected>`.
+
+        Parameters
+        ----------
+        s : int, list, optional, default : 1
+            Minimum number of edges shared by neighbors with node.
+
+        cells: boolean, optional, default: False
```

Comment:
1. Always use either `optional` or `default: XX` but never both (same for other places in this PR).
2. Its `bool`, not `boolean` in Python.

```suggestion
        cells: bool, default: False
```